src/featureflagservice/assets/css/phoenix.css

```diff
@@ -1,10 +1,19 @@
-/* Includes some default style for the starter application.
- * This can be safely deleted to start fresh.
- */
+/**
+* Copyright The OpenTelemetry Authors
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
 
-/* Milligram v1.4.1 https://milligram.github.io
- * Copyright (c) 2020 CJ Patoilo Licensed under the MIT license
- */
```

Comment:
We chatted about this in a SIG meeting a while ago and said replacing even the Google ones, which are Apache 2 would be fine.  This is MIT, so I assume it's ok, but I'm no expert specifically in this case.

Reply:
I'm a bit split. It's seems like Tristan took a pre existing template and made alterations but it only had the one initial merge and no branding changes since. 

@tsloughter do you remember working on this file / what it's initial state was or the link to the example you used? I can't find it in their GitHub rn

Reply:
From what I can tell this file has many changes from the [default v1.4.1 milligram](https://github.com/milligram/milligram/blob/main/dist/milligram.css) file available in their repo

Reply:
I'd wait for Tristan. Just to be on the safe side.

Reply:
I did not touch this as far as I can recall. Should be the default from Phoenix project generation.

Reply:
reverted to original copyright on file